microcosm_pubsub/dispatcher.py

```diff
@@ -62,6 +62,7 @@ def handle_message(self, message, bound_handlers) -> MessageHandlingResult:
                 logger=self.choose_logger(handler),
                 opaque=self.opaque,
             )
+            instance.resolve(message)
```

Comment:
It was a mistake to put the `ack/nack` calls automatically in the `invoke` call because validation failures would bypass this.